src/org/usfirst/frc4904/robot/RobotMap.java

```diff
@@ -9,13 +16,26 @@
 import edu.wpi.first.wpilibj.VictorSP;
 
 public class RobotMap {
-	public static class Port { //TODO: Correct Ports
+	public static class Port { // TODO: Correct Ports
 		public static class HumanInput {
-			public static final int joystick = 0;
+			public static final int leftStick = 0;
 			public static final int xboxController = 1;
+			public static final int rightStick = 2;
```

Comment:
this isn't that important, but maybe put the ports of the left & right stick next to each other.